content/riak/kv/3.0.15/developing/data-types/sets.md

```diff
@@ -8,7 +8,7 @@ menu:
   riak_kv-3.0.15:
     name: "Sets"
     identifier: "data_types_sets"
-    weight: 101
+    weight: 103
```

Comment:
Sets: 103 -> 104
Not sure where 101 came from, to be honest! But 104 is the right value now.